objects/Oracle.js

```diff
@@ -182,15 +182,20 @@ module.exports = {
         let returnArr = await bootstrap.LeagueObj.configGet(receivedMessage.guild.id);
         if (returnArr !== "No configs"){
             let adminPrivs = returnArr._admin;
-            if (returnArr._admin === ""){
+            if (adminPrivs.length === 0){
                 adminPrivs = "None"
             }
+            let topThreshold = 10;
+            if (returnArr._top_threshold !== undefined ){
+                topThreshold = parseInt(returnArr._top_threshold)
+            }
             const updatedEmbed = new bootstrap.Discord.MessageEmbed()
                 .setColor(bootstrap.messageColorBlue)
                 .setAuthor("Displaying information about your configurations")
                 .addFields(
-                    {name: "Minimum Games (to be appear on !top)", value: returnArr._player_threshold},
-                    {name: "Minimum Decks (to appear on !deckstats)", value: returnArr._deck_threshold},
+                    {name: "Minimum Games (Minimum number of games to appear on !top)", value: returnArr._player_threshold},
+                    {name: "Maximum Top (Maximum number of players to appear on !top)", value: topThreshold},
```

Comment:
"'Leaderboard Length' - The number of players that show up on the leaderboard (!top)\n"